Tests/Marketplace/print_testplaybook_summary.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import traceback
+import sys
+import os
+
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Utility for printing the tests summary')
+    parser.add_argument('--failed_tests_path', help='Path to the failed tests report file', required=True)
+    parser.add_argument('--succeeded_tests_path', help='Path to the succeeded tests report file', required=True)
+    return parser.parse_args()
+
+
+def read_file_contents(file_path: str) -> list | None:
+    """
+        Returns the file contents as a list of lines if the file exists, else returns None.
+    """
+    if os.path.isfile(file_path):
+        with open(file_path) as file:
+            return file.read().splitlines()
+    else:
+        logging.error(f"{file_path} does not exist.")
+    return None
+
+
+def print_test_summary(failed_tests_path: str, succeeded_tests_path: str) -> None:
+    """
+    Takes the information stored in the files and prints it in a human readable way.
+    """
+    succeeded_playbooks = read_file_contents(succeeded_tests_path)
+    failed_playbooks = read_file_contents(failed_tests_path)
+
+    # if one of the files isn't existing, we want to fail.
+    if succeeded_playbooks is None or failed_playbooks is None:
```

Comment:
Please add logging here to what happened, preferably which missing file is the culprit.

Reply:
I'm doing it in line 25
`        logging.error(f"{file_path} does not exist.")`